assets/config/postcss.config.js

```diff
@@ -5,20 +5,38 @@
  */
 module.exports = {
     plugins: [
-        // TailwindCSS
+        // TailwindCSS.
         require("tailwindcss")("./assets/config/tailwind.config.js"),
 
-        // Apply vendor prefixes for CSS features that aren't
-        // fully supported yet.
+        // Apply vendor prefixes for CSS features that aren't fully supported yet.
         require("autoprefixer")({
             overrideBrowserslist: [
                 "last 2 versions"
             ]
         }),
 
-        // Minify the CSS even further. (It works!)
-        require('cssnano')({
-            preset: 'default',
-        })
+        // Use the following plugins when not building for "development":
+        ...process.env.NODE_ENV !== "development" ? [
```

Comment:
I am I reading this correctly in that the `process.env.NODE_ENV !== "development" : [ ... ] : []` is evaluated first, and _then_ the `...` operator expands all of the elements? If so, that's super-subtle and confusing.

Would we instead just want to move this into a package level variable? Like:

```
// Include css-purgecss on non-dev builds to reduce CSS file size.
let plugins = process.env.NODE_ENV === "development" ? [] : [
];
```